x-pack/plugins/security_solution/public/timelines/components/fields_browser/helpers.tsx

```diff
@@ -141,3 +143,8 @@ export const mergeBrowserFieldsWithDefaultCategory = (
     fieldIds: defaultHeaders.map((header) => header.id),
   }),
 });
+
+export const getColumnHeaderByTimelineIdAndField = (timelineId: string, fieldId: string) =>
```

Comment:
consider renaming this to `getAlertColumnHeader`, because it only works with ` alertsHeaders`

Reply:
For now, ;)